code/index.html

```diff
@@ -8,11 +8,59 @@
   </head>
   <body>
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+      <ul class="navigation-list">
+        <li><a href="articles.html">Articles</a></li>
+        <li><a href="about.asp">About</a></li>
+        <li><a href="contact.asp">Contact</a></li>
+      </ul>
     </header>
 
-    <main class="">
-      <!-- The main container to use as the flex parent for your card layout, the main content of the site should be in here.  -->
-    </main>
+    <img src="images/comics.jpg" alt="Comic books" class="top-image" />
+
+    <h1>Articles</h1>
+
+    <ul class="article-container">
+      <li class="article">
+        <img src="images/anime.jpg" alt="Anime" class="article-image" />
+        <h2>Article Nr. 1</h2>
+      </li>
+
+      <li class="article">
+        <img src="images/book.jpg" alt="Book" class="article-image" />
+        <h2>Article Nr. 2</h2>
+      </li>
+
+      <li class="article">
+        <img src="images/figurine.jpg" alt="Figurine" class="article-image" />
+        <h2>Article Nr. 3</h2>
+      </li>
+
+      <li class="article">
+        <img src="images/food.jpg" alt="Food" class="article-image" />
+        <h2>Article Nr. 4</h2>
+      </li>
+
+      <li class="article">
+        <img src="images/octopus.jpg" alt="Octopus" class="article-image" />
+        <h2>Article Nr. 5</h2>
+      </li>
+
+      <li class="article">
+        <img src="images/panel.jpg" alt="Panel" class="article-image" />
+        <h2>Article Nr. 6</h2>
+      </li>
+    </ul>
+
+    <div class="about-container">
+      <h1>About</h1>
+      <img src="images/shop.jpg" alt="Shop" class="about-image" />
```

Comment:
This image is veeery big, maybe style it smaller? It's causing a major side scroll to your page 😅